src/main/resources/db/changelog/changes/v1.1.0/insert-organizations-contact-info-entity-type.xml

```diff
@@ -296,15 +295,67 @@
                       "property": "description",
                       "dataType": { "dataType": "stringType" },
                       "queryable": true,
-                      "valueGetter": "( SELECT array_agg(elems.value ->> 'description') FROM jsonb_array_elements(org.jsonb->'urls') AS elems)",
                       "filterValueGetter": "( SELECT array_agg(lower(elems.value ->> 'description')) FROM jsonb_array_elements(org.jsonb -> 'urls') AS elems)",
                       "valueFunction": "lower(:value)"
+                    },
+                    {
+                      "name": "category_ids",
+                      "property": "categories",
+                      "dataType": {
+                        "dataType": "arrayType",
+                        "itemDataType": { "dataType": "rangedUUIDType" }
+                      },
+                      "queryable": false,
+                      "filterValueGetter": "( SELECT array_agg(lower(elems#>>'{}')) FROM jsonb_array_elements(org.jsonb->'urls') AS url_obj, jsonb_array_elements(url_obj->'categories') as elems )"
+                    },
+                    {
+                      "name": "category_names",
+                      "property": "categoriesNames",
+                      "dataType": {
+                        "dataType": "arrayType",
+                        "itemDataType": { "dataType": "stringType" }
+                      },
+                      "queryable": true,
```

Comment:
I don't think our existing ID column work supports nested things like this, especially this deeply nested. So I'm thinking we should leave as just having "names" queryable? We lose some resiliency if the underlying category is renamed, but otherwise it should be fine.

Reply:
yeah, I don't think it would be worth it to rework all the other logic just for one field, so I like this idea